recipes/pypy/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "pypy2.7" %}
+{% set version = "5.10.0" %}
+{% set hash = "c966124497ba8728654ce1161fa4c46b035ff30f289be70960f58292e5897cc8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://bitbucket.org/squeaky/portable-pypy/downloads/pypy-5.10.0-linux_x86_64-portable.tar.bz2
```

Comment:
Shouldn't we really be building this from scratch rather than copying a pre-built source?

Reply:
This is a prebuilt source

Reply:
@patricksnape not necessarily.  It is less CI burden to go from the prebuilt source. Sometimes there could be linking problems with prebuilt source or too many bundled dependencies which is why we'd go with building ourselves.

Reply:
I tried to build pypy from scratch once and it took more than an hour. I don't think our CIs can handle that at the moment.